include/class.team.php

```diff
@@ -88,6 +88,18 @@ function getMembers() {
         return $this->_members;
     }
 
+    function getMembersForAlerts() {
+        if (!isset($this->_members)) {
+            $this->_members = array();
```

Comment:
This will override $this->_members used in `getMembers()` -  whichever method gets called first will set the variable and subsequent calls may result in wrong members being returned.